### ISSUE TITLE
latest dev build slows processing / only one instance works

### ISSUE THREAD

**Issue by liuboto** [OpenEndedGroup/Field#14]
the latest dev build 10/15/12 makes processing sketches load really slow compared to previous versions. moreover, only the first sketch with processing works. the second one no longer works. this is similar to #3 but worse, because in #3 the second processing instance broke only after closing the first instance. it is problematic because the default behavior opens default.sketch which may have a processing box in it. a quick solution that does not involve command line starting.

right now if one double-clicks a field sketch in the finder, field also opens default.sketch. it would be nice to have field only open the actual clicked sketch as a temp workaround.

attached bug report

https://dl.dropbox.com/u/4220119/field/fieldprocessingbugginess.txt.zip
